index.js

```diff
@@ -258,6 +258,13 @@ class Help {
    */
 
   argumentDescription(argument) {
+    const extraInfo = [];
```

Comment:
`extraInfo` appears to have only 0 or 1 elements.
Does it need to be an array?

Reply:
Fair question. No, does not need to be an array currently. However, I left in the array as planning to add choices, like `optionDescription` which I copied the code from.